src/ttt/core.clj

```diff
@@ -14,18 +15,21 @@
         (fn [current-state]
            (merge current-state {:board board :players players :current-player (first players)}))))
 
-(defn play 
+(defn play
   []
-  (ui/print-board (:board @game-state))
-  (set-game-state 
-   (board/make-move (:board @game-state) (player/get-move) (:marker (:current-player @game-state))) 
-   (player/switch-players (:players @game-state)))
-  (ui/print-board (:board @game-state)))
+  (loop [state @game-state]
```

Comment:
Now that you have a loop, do you think you still need the game state atom?

Reply:
doesn't look like it! I refactored it to take a starting game state at the beginning when `play` is called, and i just create a new state each time it finishes the loop.